src/aws/dynamodb-utils.ts

```diff
@@ -0,0 +1,34 @@
+import AWS from 'aws-sdk';
+import {
+  BatchWriteItemInput,
+  BatchWriteItemOutput,
+  BatchWriteItemRequestMap,
+  ItemList,
+  PutItemInput,
+  PutItemInputAttributeMap,
+  PutItemOutput,
+  QueryInput,
+} from 'aws-sdk/clients/dynamodb';
+
+const ddb = new AWS.DynamoDB.DocumentClient();
+
+export const saveItem = async (table: string, item: PutItemInputAttributeMap): Promise<PutItemOutput> => {
+  const params: PutItemInput = {
+    TableName: table,
+    Item: item,
+  };
+  return await ddb.put(params).promise();
```

Comment:
theoretically, you'd just want to return the promise() instead of awaiting...

Reply:
yeah, good catch copy pasta from older code. updates it on other pr i have open that' i'm still working on